src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java

```diff
@@ -414,6 +421,10 @@ protected void onStartup() {
                     "if no sample file (--sample-file) is provided.");
         }
 
+        OptionalLong optionalMaximumAlternateAlleles =
```

Comment:
I'm not opposed to OptionalLong, but it seems out of character with respect to the rest of the code which is firmly in classic Java camp.

Reply:
Hmm not sure what "classic Java" is, but`Optional`s are all over GATK. 🙂 I can make this an `Optional<Long>` if that seems less jarring.